docs/pages/router/reference/api-routes.mdx

```diff
@@ -442,6 +442,62 @@ Start the server with `node` command:
 
 </Step>
 
+### Bun
+
+<Step label="1">
+
+Export the website for production:
+
+<Terminal cmd={['$ npx expo export -p web']} />
+
+</Step>
+
+<Step label="2">
+
+Write a server entry file that serves the static files and delegates requests to the server routes:
+
+```ts server.ts
+import { createRequestHandler } from '@expo/server/adapter/bun';
+import dotenv from 'dotenv';
+
+import { websocket } from './websocket';
+dotenv.config();
+
+const CLIENT_BUILD_DIR = `${process.cwd()}/dist/client`;
+const SERVER_BUILD_DIR = `${process.cwd()}/dist/server`;
+const handleRequest = createRequestHandler({ build: SERVER_BUILD_DIR });
+
+const port = process.env.PORT || 3000;
+
+Bun.serve({
+  port: process.env.PORT || 3000,
+  async fetch(req) {
+    const url = new URL(req.url);
+    console.log('Request URL:', url.pathname);
+
+    const staticPath = url.pathname === '/' ? '/index.html' : url.pathname;
```

Comment:
Is the list of files known ahead of time? If so, I recommend using the `routes` option in Bun.serve passing Bun.file objects for each so you can delete this code

Reply:
Oh this is documentation not implementation, hm

Reply:
We've got an internal task that tracks converting our routes format to Bun's format. But we can come back to that separately. No need to address this straight away in this PR ✌️

Tracked internally and on our radar though